website/source/docs/providers/azurerm/r/template_deployment.html.markdown

```diff
@@ -98,6 +98,7 @@ The following arguments are supported:
 The following attributes are exported:
 
 * `id` - The Template Deployment ID.
+* `outputs` - A map of supported scalar output types returned from the deployment (currently, Azure template outputs of type String, Int and Bool are supported, and are converted to strings - others will be ignored).
```

Comment:
> currently, Azure template outputs of type String, Int and Bool are supported, and are converted to strings - others will be ignored

Is it worth changing `Azure template outputs` -> `AzureRM Template Deployment Outputs of..` for consistency here?